greeneeds/projects/models.py

```diff
@@ -0,0 +1,59 @@
+from django.db    import models
+
+from users.models import User
+from core.models  import TimeStampModel
+
+class Project(TimeStampModel):
+    user            = models.ForeignKey(User, on_delete=models.CASCADE)
+    title           = models.CharField(max_length=70)
+    shortcut_title  = models.CharField(max_length=50)
+    summary         = models.CharField(max_length=100)
+    price           = models.DecimalField(max_digits=7, decimal_places=2)
+    thumbnail       = models.URLField(max_length=150)
+    pk_uri          = models.CharField(max_length=50)
+    target_amount   = models.DecimalField(max_digits=9, decimal_places=2)
+    start_datetime  = models.DateField()
+    end_datetime    = models.DateField()
+    pay_end_date    = models.DateField()
+    settlement_date = models.DateField()
+    introduction    = models.CharField(max_length=200)
+    budget_plan     = models.CharField(max_length=200)
+    organizations   = models.ManyToManyField("Organization", through="ProjectOrganization", related_name="project")
+
+    class Meta:
+        db_table = "projects"
+
+class Category(models.Model):
+    project   = models.ForeignKey(Project, on_delete=models.CASCADE)
+    name      = models.CharField(max_length=50)
+    image_url = models.URLField(max_length=150)
+
+    class Meta:
+        db_table = "categories"
+
+class ProjectImage(models.Model):
+    project   = models.ForeignKey(Project, on_delete=models.CASCADE)
+    image_url = models.URLField(max_length=150)
+
+    class Meta:
+        db_table = "project_images"
+
+class Tag(models.Model):
+    project = models.ForeignKey(Project, on_delete=models.CASCADE)
+    tag     = models.CharField(max_length=50)
+
+    class Meta:
+        db_table = "tags"
+
+class Organization(models.Model):
+    name = models.CharField(max_length=50)
+
+    class Meta:
+        db_table = "organizations"
+
+class ProjectOrganization(models.Model):
+    project      = models.ForeignKey(Project, on_delete=models.CASCADE)
+    organization = models.ForeignKey(Organization, on_delete=models.CASCADE)
+
+    class Meta:
+        db_table = "project_organizations"
```

Comment:
다대다의 관계 테이블은 두 자원을 나타내는 단어들을 모두 복수로 많이 표현 합니다.

```suggestion
        db_table = "projects_organizations"
```